appengine/test/test_helper.rb

```diff
@@ -0,0 +1,73 @@
+# A set of helpful methods for running tests.
+
+class TestHelper
+
+  def initialize(opts)
+    @verbose = opts[:verbose]
```

Comment:
Reworked in https://github.com/GoogleCloudPlatform/ruby-docker/pull/24. It always outputs, but simplified (similar to the behavior of rake's `sh`)